src/JMS/Serializer/Expression/ExpressionEvaluator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace JMS\Serializer\Expression;
+
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+
+class ExpressionEvaluator implements ExpressionEvaluatorInterface
```

Comment:
Is there a need to support another evaluator? Even if there were one, we already have the strategy interface which could be implemented, no?